Header header
Float32MultiArray x
Float32MultiArray y